src/ipsis/course-merge/CourseMergeLogDetailEntity.js

```diff
@@ -14,8 +14,8 @@ export class CourseMergeLogDetailEntity extends Entity {
 		return this._entity?.getSubEntitiesByClass(Classes.ipsis.sisCourseMerge.sourceLog);
 	}
 
-	user() {
-		return this._entity?.getSubEntityByClass(Classes.ipsis.sisCourseMerge.logUser);
+	userHref() {
+		return this._entity?.getSubEntityByClass(Classes.ipsis.sisCourseMerge.logUser).getLinkByRel(Rels.ipsis.sisCourseMerge.logUser).href;
```

Comment:
is it worth putting the null checks in this case?
`this._entity?.getSubEntityByClass(Classes.ipsis.sisCourseMerge.logUser)?.getLinkByRel(Rels.ipsis.sisCourseMerge.logUser)?.href`

Reply:
done